web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -20,9 +20,31 @@
             style="text-align: center;"
             outlined
           >
+            <v-tooltip
+              top
+            >
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  v-if="currentDandiset !== null"
+                  icon
+                  small
+                  v-bind="attrs"
+                  v-on="on"
+                  @click="currentDandiset ? copy(currentDandiset.dandiset.identifier) : ''"
```

Comment:
i think we want to do this for copying the identifer, as that is the accession id for the dandiset.

```suggestion
                  @click="currentDandiset ? copy('DANDI:' + currentDandiset.dandiset.identifier) : ''"
```

Reply:
Furthermore,  I don't like the idea of using a ternary expression as an if statement to perform an action. I think we should just have `@click="copy"` here, and move the if condition into that function.

Reply:
updated in 6d3b2bfa463bce605d92904392f99d25f6d6ddf1